client/src/app/bookings/layout.tsx

```diff
@@ -36,18 +47,23 @@ const InnerBookingLayout = ({ children }: IBookingLayout) => {
         </div>
       </div>
       <span className="relative z-[11]">
-        <PolicyTabs />
+        <PolicyTabs policiesArray={PoliciesContent} />
       </span>
       <Footer />
     </>
   )
 }
 
-export default function BookingLayout({ children }: IBookingLayout) {
+export default function BookingLayout({
```

Comment:
I think the problem is this default export shouldn't be changed because we are using `BookingPolicyStorage` now

Reply:
need to undo all changes to this, have a look at the check builds workflow